packages/web-components/src/components/cta-section/cta-section.scss

```diff
@@ -13,11 +13,15 @@
   @include cta-section--pattern;
 
   display: block;
+
+  ::slotted([slot='buttons']) {
+    margin-top: 1rem;
+  }
 }
 
 .#{$dds-prefix}--content-block__copy {
   display: inline-block;
-  font-size: 1.5rem;
+  font-size: calc(1.25rem + 0 * ((100vw - 20rem) / 62));
```

Comment:
Wondering where this change comes from...?

Reply:
The font size change comes from the `React` version, I think it was changed from 1.5rem to that calculation recently. Comes from the `_cta-section.scss` under the `styles` package.